src/base/rss/rss_feed.cpp

```diff
@@ -50,21 +51,30 @@
 #include "rss_article.h"
 #include "rss_session.h"
 
-const QString Str_Url(QStringLiteral("url"));
-const QString Str_Title(QStringLiteral("title"));
-const QString Str_LastBuildDate(QStringLiteral("lastBuildDate"));
-const QString Str_IsLoading(QStringLiteral("isLoading"));
-const QString Str_HasError(QStringLiteral("hasError"));
-const QString Str_Articles(QStringLiteral("articles"));
+const QString KEY_UID(QStringLiteral("uid"));
```

Comment:
It was my experiments with the naming of global constants. I rejected it in the end. It is better to stick to the existing scheme so I renamed it.